pkg/symbolizer/symbolizer.go

```diff
@@ -55,22 +70,64 @@ type DebugInfoFetcher interface {
 
 func New(
 	logger log.Logger,
+	reg prometheus.Registerer,
 	metastore pb.MetastoreServiceClient,
 	debuginfo DebugInfoFetcher,
 	symbolizer *symbol.Symbolizer,
 	debuginfodCacheDir string,
 	debuginfoCacheDir string,
 	batchSize uint32,
 ) *Symbolizer {
-	return &Symbolizer{
+	attemptsTotal := promauto.With(reg).NewCounter(
+		prometheus.CounterOpts{
+			Namespace: "parca",
```

Comment:
I think we should omit `Namespace: "parca"`. This will result in the metric name being `parca_symbolizer_symbolication_attempts_total`. We don't use `parca` as a prefix elsewhere. 
Also, I don't mind collecting everything in the name. The same goes for metrics.

Reply:
Alright, that's done :)